instance/models/mixins/openedx_config.py

```diff
@@ -77,6 +76,13 @@ def _get_configuration_variables(self):
                     # because we also add other extended checks
                     "openedx.core.djangoapps.heartbeat.default_checks.check_celery",
                 ],
+                # rewrite marketing links:
+                "MKTG_URL_OVERRIDES": {
+                    # remove blog link - not supported yet
+                    "BLOG": "",
+                    # use different contact page
+                    "CONTACT": "/contact",
```

Comment:
Is this variable required now, given that CONTACT_US_CUSTOM_LINK is the one being picked for the url? If not, we can remove this (if required - what happens if both the variables have conflicting values?)

Reply:
That's right, so:
1. `CONTACT_US_CUSTOM_LINK` is used for rendering footer link
2. `MKTG_URL_OVERRIDES['CONTACT']` used for email here: [djangoapps/student/views/management.py#L749](https://github.com/edx/edx-platform/blob/98af9ce4185bde0a38030923a88403a101e011ab/common/djangoapps/student/views/management.py#L749 ) and for home page footer (as I understand for custom theme: [red-theme/lms/templates/footer.html#L47](https://github.com/edx/edx-platform/blob/752b3072c4854752a0e6f4827c959917b555992f/themes/red-theme/lms/templates/footer.html#L47))

So we have to set `MKTG_URL_OVERRIDES['CONTACT']` this key, edx may also use it somewhere else as well.